### ISSUE TITLE
Error on loadFunc with no pvp talents selected

### ISSUE THREAD

**Issue by emptyrivers** [WeakAuras/WeakAuras2#682]
**Describe the bug**
Reported on Discord
```
TigewToday at 7:28 PM
So I'm at kinda a loss for what to try, WAs stopped loading when they should. I try to remove and re-add the wa that should be loading but that doesn't work, it doesn't even fully import aside from the group name and that's it, downgrade from alpha to current release. try again same result broken import, delete wa saved option and bak and cache and only have wa enabled, same result as before what should I do?

riversToday at 7:31 PM
!bugsack

WeakAurasBotBOTToday at 7:31 PM
Install both of these addons:
https://mods.curse.com/addons/wow/BugSack
https://mods.curse.com/addons/wow/bug-grabber
Then try again. Report back any error messages you get, and we will be able to give more specific advice.

TigewToday at 7:33 PM
Fixed it immediately, turns out WA loses it's mind if you don't have pvp talents selected
```

**Do you have an error log of what happened?**
```
257x WeakAuras\Prototypes.lua:496: Usage: GetTalentInfoByID(talentID, specGroupIndex [, isInspect, inspectUnit])
[C]: in function `GetPvpTalentInfoByID'
WeakAuras\Prototypes.lua:496: in function `CheckPvpTalentByIndex'
[string "--[[ Error in ' Unknown' ]]return function(..."]:3: in function `loadFunc'
WeakAuras\WeakAuras-2.7.2-24-gca5050b.lua:1688: in function `ScanForLoads'
WeakAuras\WeakAuras-2.7.2-24-gca5050b.lua:1775: in function <WeakAuras\WeakAuras.lua:1773>

Locals:
(*temporary) = nil
```

**Comment by emptyrivers** [comment]
Eh, this was on 2.7.2, I don't know if the issue persists in later versions.

**Comment by emptyrivers** [comment]
Confirmed that the issue persists on cb96f590fb456976f579297a6b336a2f1384e0ba

**Comment by emptyrivers** [comment]
Fixed in 2.7.4
